File: 2177-find-three-consecutive-integers-that-sum-to-a-given-number/2177-find-three-consecutive-integers-that-sum-to-a-given-number.py
class Solution:
    def sumOfThree(self, num: int) -> List[int]:
        a=(num-3)/3
        l=[]
        if(a==int(a)):
            l=l+[int(a)]+[int(a+1)]+[int(a+2)]
        return(l)
        
        

File: 2177-find-three-consecutive-integers-that-sum-to-a-given-number/2177-find-three-consecutive-integers-that-sum-to-a-given-number.cpp
class Solution {
public:
    vector<long long> sumOfThree(long long num) {
        vector<long long int> v;
        double b;
        b=(num-3.0000)/3;
        long long int c=b;
        if(c==b)
        {
            v.push_back(c);
            v.push_back(c+1);
            v.push_back(c+2);
        }
        return(v);
    }
};